app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -0,0 +1,100 @@
+SUMMARY="A scientific editing platform designed to create beautiful technical documents"
+DESCRIPTION="TeXmacs provides a unified and user friendly framework for editing \
+structured documents with different types of content: text, mathematics, graphics, \
+interactive content, slides, etc.\
```

Comment:
Remove the backslash here, that way the new paragraph starts on a new line in HaikuDepot

Reply:
done.